federatedscope/cl/model/SimCLR.py

```diff
@@ -0,0 +1,235 @@
+import torch
```

Comment:
This file looks like a copy from https://github.com/akhilmathurs/orchestra/blob/main/models.py.

Please consider the copyright issues.